### PR TITLE
Launcher: improve demos presentation with new fields provided by the RRM

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: apps/Launcher/qml/DemoLauncher/DemoLauncher.qml [hunk 1]
```diff
@@ -46,46 +44,53 @@ Rectangle {
...
-
-                    gradient: Gradient {
-                        GradientStop { position: 0.0; color: Style.placeholderTopColor }
-                        GradientStop { position: 1.0; color: Style.placeholderBottomColor }
-                    }
-                    Image {
-                        id: image
-                        anchors.fill: parent
+            width: parent.width
+            height: 1.2 * itemSize
```
Comment (favreau) [inline apps/Launcher/qml/DemoLauncher/DemoLauncher.qml]: Maybe replace 1.2 by some constant with a more explicit name?

File: apps/Launcher/qml/DemoLauncher/DemoLauncher.qml [hunk 2]
```diff
@@ -100,7 +105,7 @@ Rectangle {
         Text {
             id: titleText
             anchors.fill: parent
-            anchors.margins: 0.1 * height
+            anchors.margins: 0.1 * parent.height
```
Comment (favreau) [inline apps/Launcher/qml/DemoLauncher/DemoLauncher.qml]: idem with 0.1 and 0.2?

**Comment by rdumusc** [comment]
You're right, there are many magic numbers in these UI files, and I was a bit lazy about improving the situation ;-) But I tried to remove most of them from the DemoLaucher

**Comment by favreau** [comment]
Excellent, I was not sure if this was a standard thing to do in QML but it's looking much better. Thanks for that.
+1

**Pull request merged**